content/contributor/tutorials/how-to-create-new-entity-type.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Create a new Entity in the Content Server"
+sidebartitle: New entity type
+url: "/contributor/tutorials/how-to-create-new-entity-type"
+---
+
+### When is needed to create a new entity?
+First, check if an existing entity can solve the needed use case to avoid over engineering a solution.
+If none of the existing entity types defined in https://github.com/decentraland/common-schemas/blob/main/src/platform/entity.ts#L13 satisfies the requirements, then you may need to create a new entity type.
+
+### First answer:
+- Is the new entity an NFT Collection?
+- How will the ownership be validated?
+- Which will be the size of it?
+- What is the amount entities that will be created?
```

Comment:
@pedrotambo please add the considerations here. I can ask myself those questions but there is no framework to decide on the answer